lib/ui/painting.dart

```diff
@@ -1568,6 +1568,15 @@ class Paint {
   }
 }
 
+/// The color space that an [Image] uses.
+enum ColorSpace {
```

Comment:
Consider documenting the support for different values across our various backends

Reply:
I don't think we want to do that because regardless of the backend the colorSpace property gives valid results, so it's inconsequential what the backend is and discussing it here could confuse and get stale easily.

Reply:
I added a note to `ui.Image.colorSpace` that wide gamut images may still return sRGB on backends that don't support wide gamut colors.  I think that addresses your concerns here and won't ever get stale.

Reply:
In general, if someone is reading the documentation, it's because they had a question. What could that question be?

Maybe here the question is "what is a color space", or "how do i change the color space that an image is using", or "how do i pick which color space to specify". Might be worth answering all of these here.

Reply:
Okay, i've elaborated a bit more with my last commit.

Reply:
Its kind of a recusive definition though?

I don't think we need a whole wikipedia article, but short explanation of what the thing is and more specific definition of how its used:

"This value can help you decide which [ImageByteFormat] to use with [Image.toByteData] and is the result of [Image.colorSpace]"

This is sort of vague, how do I use it to decide what?

Reply:
I've reworded it to make it more clear it is a collection of colors.  Good point on the toByteData usage.  I made that more clear.

Reply:
might be worth just linking to wikipedia in a "See also" block, too.

Reply:
Link added.